crates/cext/cbindgen.toml

```
@@ -10,6 +10,13 @@ after_includes = """
     #include <Python.h>
 #endif
 
+#define QISKIT_VERSION_MAJOR 2
+#define QISKIT_VERSION_MINOR 0
+#define QISKIT_VERSION_PATCH 2
```

This will be included in the 2.1.0 release 🙂 2.0.2 is just a bugfix release without any new features (like this one)
```suggestion
#define QISKIT_VERSION_MINOR 1
#define QISKIT_VERSION_PATCH 0
```